it/v2/connectors/PersistenceConnectorSpec.scala

```diff
@@ -81,8 +81,8 @@ class PersistenceConnectorSpec
     with ScalaCheckDrivenPropertyChecks
     with CommonGenerators {
 
-  lazy val appConfig: AppConfig = app.injector.instanceOf[AppConfig]
-
+  lazy val appConfig: AppConfig                           = app.injector.instanceOf[AppConfig]
+  lazy val messageTypeGen                                 = Gen.oneOf(Seq(MessageType.DeclarationAmendment))
```

Comment:
If this will over produce a DeclarationAmendment, why is it a generator?

Reply:
done the changes